audit/audit-gcp.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# set -x -e
+CNCF_GCP_ORG=758905017065
+
+# gcloud organizations describe $CNCF_GCP_ORG 2>&1
+# ERROR: (gcloud.organizations.describe)
+# User [hh@ii.coop] does not have permission to access organization []
+
+format=json
+echo "# Auditing CNCF CGP Org: ${CNCF_GCP_ORG} #"
+gcloud iam roles list --organization=$CNCF_GCP_ORG --format=$format \
+       > cncf-org.roles.$format
+gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
+       > cncf-org.policy.$format
+echo "## Iterating over Projects ##"
+gcloud projects list \
+       --filter "parent.id=$CNCF_GCP_ORG" \
+       --format "value(name, projectNumber)" \
+    | while read PROJECT NUM; do \
+    export CLOUDSDK_CORE_PROJECT=$PROJECT
+    echo "### Auditing Project: ${PROJECT} ###"
+    mkdir -p $PROJECT
```

Comment:
Since this assumes CWD is the script's root, maybe we should check that and print a warning if it is not?